addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -46,17 +48,25 @@ var ForumShare = publicWidget.registry.socialShare.extend({
      * @private
      */
     _render: function () {
-        var $question = this.$('article.question');
+        var questionEl = this.el.querySelector('article.question');
```

Comment:
```suggestion
        var questionEl = this.el.querySelector("article.question");
```
We use double quotes in Javascript now.

Reply:
@SergeBayet  Thank you for the review. I have fixed the JS linting issues as suggested.